demo/README.md

```diff
@@ -9,125 +9,13 @@ To run the completed project in this folder, you need the following:
-    | `AzureADMyOrg` | Only users in your Microsoft 365 organization |
-    | `AzureADMultipleOrgs` | Users in any Microsoft 365 organization (work or school accounts) |
-    | `AzureADandPersonalMicrosoftAccount` | Users in any Microsoft 365 organization (work or school accounts) and personal Microsoft accounts |
-    | `PersonalMicrosoftAccount` | Only personal Microsoft accounts |
-
-1. Copy the **Client ID** and **Auth tenant** values from the script output. You will need these values in the next step.
-
-    ```powershell
-    SUCCESS
-    Client ID: 2fb1652f-a9a0-4db9-b220-b224b8d9d38b
-    Auth tenant: common
-    ```
-
-#### Configure app-only auth (PowerShell)
-
-> **Note:** This section requires a work/school account with the Global administrator role. You only need to complete these steps if you plan on using the app-only portions of this sample.
-
-1. Run the [UpdateAppForAppOnlyAuth.ps1](UpdateAppForAppOnlyAuth.ps1) file with the following command, replacing *&lt;your-client-id&gt;* with your client ID.
-
-    ```powershell
-    .\UpdateAppForAppOnlyAuth.ps1 -AppId <your-client-id> -GraphScopes "User.Read.All"
-    ```
-
-1. Copy the **Tenant ID** and **Client secret** values from the script output. You will need these values in the next step.
-
-    ```powershell
-    SUCCESS
-    Tenant ID: a795ad0f-7d82-4a3b-a2c0-0713ec72ade7
-    Client secret: 2jv7Q~8eiOd_QafJ.....
-    Secret expires: 2/16/2024 9:32:09 PM
-    ```
-
-## Configure the sample
-
-1. Open [appsettings.json](./GraphTutorial/appsettings.json) and update the values according to the following table.
-
-    | Setting | Value |
-    |---------|-------|
-    | `clientId` | The client ID of your app registration |
-    | `tenantId` | The tenant ID of your organization (only needed if doing app-only) |
-    | `authTenant` | If you chose the option to only allow users in your organization to sign in, change this value to your tenant ID. Otherwise leave as `common`. |
-
-1. Initialize the [.NET development secret store](https://docs.microsoft.com/aspnet/core/security/app-secrets) by opening your CLI in the directory that contains **GraphTutorial.csproj** and running the following command.
-
-    ```Shell
-    dotnet user-secrets init
-    ```
-
-1. Add your client secret to the secret store using the following command, replacing *&lt;client-secret&gt;* with your client secret.
-
-    ```Shell
-    dotnet user-secrets set settings:clientSecret <client-secret>
-    ```
-
-    > **Note:** The .NET Secret Manager is only available during development. Production apps should store client secrets in a secure store, such as [Azure Key Vault](https://docs.microsoft.com/azure/key-vault/general/overview).
+By default this sample disables the app-only authentication portion. This is because this portion requires access to a work/school account with administrative privileges. For instructions to configure and enable app-only authentication, see [APP-ONLY-CONFIG.md](APP-ONLY-CONFIG.md).
```

Comment:
```suggestion
By default, this sample disables the app-only authentication portion. This is because app-only authentication requires access to a work/school account with administrative privileges. For instructions to configure and enable app-only authentication, see [APP-ONLY-CONFIG.md](APP-ONLY-CONFIG.md).
```